### ISSUE TITLE
Strings vs Enums

### ISSUE THREAD

**Issue by fitzyjoe** [TeslaGov/clarakm-aws-events#1]
I see that an enum was used for Topic, but static strings were used for subjects... why not use enums for both?

**Comment by kevinmichaelchen** [comment]
I can go either way. Don't see why we couldn't use Strings. There's a `formatWithSuffix` method on the enum that would have to be refactored, but it could work.

**Comment by fitzyjoe** [comment]
What's the suffix for?  the environment?

**Comment by crain9412** [comment]
Yeah for factory we'll probably have a set of local queues/topics each and then maybe an integ one.  On prod there will probably be no suffix.

**Comment by kevinmichaelchen** [comment]
@fitzyjoe The suffix is so our local builds don't all write to the same topics and queues, e.g., my local Rampart will write to "rampart-person-topic-kevin", whereas yours will write to "rampart-person-topic-joe".
